webpack/assets/javascripts/react_app/components/common/forms/Form.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Actions from './Actions';
+import Alert from '../Alert';
+
+export default ({
+  className = 'form-horizontal well',
```

Comment:
... because the ruby form helpers already do it that way.

https://github.com/theforeman/foreman/blob/16bd5715afbfdbea20f653435973908d395a8f7a/app/helpers/form_helper.rb#L284-L291

Reply:
I don't see patternfly using `.well` anywhere on their [forms](https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/form.html).  Can we remove this from both ruby form helpers and from this implementation?

Reply:
@ohadlevy thoughts on this?

Reply:
@waldenraines TBH its been like that every since we started with bootstrap, I don't have a strong opinion one way or another, but i don't think it should be done in this pr, if we want to change it, then it should be across the app and not only in this PR.

Reply:
I'm okay with that, see http://projects.theforeman.org/issues/21171